stripe/src/main/java/com/stripe/android/view/CheckoutActivity.kt

```diff
@@ -28,6 +28,12 @@ internal class CheckoutActivity : AppCompatActivity() {
         }
 
         setupBottomSheet()
+
+        // TODO: Add loading state
+        supportFragmentManager
+            .beginTransaction()
+            .replace(viewBinding.fragmentContainer.id, CheckoutPaymentMethodsListFragment())
+            .commitAllowingStateLoss()
```

Comment:
ooc, can you use the ktx functions?

https://developer.android.com/kotlin/ktx/extensions-list#for_fragmenttransaction

Reply:
ah, yes